scripts/run

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euo pipefail
+
+DATABASE_URL="${DATABASE_URL:-""}"
+
+if [ -z "${DATABASE_URL}" ]; then
+  if [ -f .env ]; then
+    source .env
+  fi
+fi
+
+if [ -z "${DATABASE_URL}" ]; then
+  echo "DATABASE_URL not set." >&2
+  exit 1
+fi
+
+if [ ! -f "database/src/schema.rs" ]; then
+  echo "Database schema does not exist, run scripts/setup before attempting to run the tests." >&2
```

Comment:
This warning is correct and `scripts/setup` should likely continue to do the schema generation, but in retrospect `build.rs` could also just do the schema generation as well, and avoid having the user forced to take action.

Reply:
Investigated this further, and because db stuff is broken out into its own crate, its own `build.rs` would need to run first in order to get the crate to build. Unfortunately, SQLite paths would likely be qualified in the root crate, not the dependent crate, so any `DATABASE_URL` that exists that would allow the schema generation to proceed would need to be requalified within the context of the `database` subdirectory, at which point this would all start to feel like it's doing too much work / assuming too much on what's inside that env var.

Just leaving things as is for now.